tests/test_basic_twoword_classifier.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import torch
+from scripts import BasicTwoWordClassfier
+import unittest
+
+
+class BasicTwoWordClassifierTest(unittest.TestCase):
+    """
+    this class tests the BasicTwoWordClassifier
+    This test suite can be ran with:
+        python -m unittest -q tests.BasicTwoWordClassifierTest
+    """
+
+    def setUp(self):
+        self.word1 = torch.from_numpy(np.array([[1, 0, 0]], dtype='float32'))
+        self.word2 = torch.from_numpy(np.array([[0, 1, 0]], dtype='float32'))
+        self.inputdim = 6
+        self.hiddendim = 6
+        self.labels = 2
+
+    def test_forward(self):
+        """
+        tests the classifier implemented in BasicTwoWordClassifier and the overridden method "forward"
+        prints the classifier to check whether the right number of layers exist and the right dimensions are included
+        """
+        classifier = BasicTwoWordClassfier(input_dim=self.inputdim, hidden_dim=self.hiddendim, label_nr=self.labels)
+        classifier.forward(self.word1, self.word2)
+        print(classifier)
```

Comment:
I would say: batter look at the result that you get from "forward". It should be a matrix [batchsize, label_size] in your case, because batch_size is 1 a vector of shape [1, 2]. The test does not really test something because you manually have to look at the print statement and see whether the shape is correct. instead you can store the result from forward in a variable and call the np.assert.. method to ensure that the shape of your result has the shape you want. I also think that this test is enough for now. we can add more tests later